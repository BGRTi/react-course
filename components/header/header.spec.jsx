import React from 'react';
import { shallow } from 'enzyme';

import Header from './Header';

function setup() {
  const wrapper = shallow(<Header />);
  return { wrapper };
}

describe('HelloFunctionalComponent Test Suite', () => {
  it('render a Header', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });

  it('Should have an image', () => {
    const { wrapper } = setup();
    expect(wrapper.find('.main-header').exists()).toBe(true);
  });
});