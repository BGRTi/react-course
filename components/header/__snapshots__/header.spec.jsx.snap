// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HelloFunctionalComponent Test Suite render a Header 1`] = `ShallowWrapper {}`;
